backend/src/main/java/com/ody/meeting/dto/response/MeetingFindByMemberResponse.java

```diff
@@ -0,0 +1,55 @@
+package com.ody.meeting.dto.response;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public record MeetingFindByMemberResponse(
+        @Schema(description = "약속 아이디", example = "1")
+        Long id,
+
+        @Schema(description = "약속 이름", example = "조조와 저녁 초밥")
+        String name,
+
+        @Schema(description = "약속 인원 수", example = "2")
+        int mateCount,
+
+        @Schema(description = "약속 날짜", example = "2024-09-10")
+        String date,
+
+        @Schema(description = "약속 시간", example = "13:30")
+        String time,
```

Comment:
```suggestion
        @Schema(description = "약속 날짜",  type = "string", example = "2024-09-10")
        LocalDate date,

        @Schema(description = "약속 시간", type = "string", example = "13:30")
        LocalTime time,
```
아직 일어나기 전이라 2번에 대해 자세히는 모르겠지만
예측하기로는 시간, 날짜 타입을 그대로 가져가고 Schema에 type 옵션을 string으로 주면 좀더 간편하게 해결될 것 같아요 😄

Reply:
description을 이어서 작성해두었어요. 저는 type 지정으로 해결할 수 없는 부분이라 판단했는데 추가된 글을 보고 카키의 생각을 남겨주세요! 😊

Reply:
```suggestion
        @Schema(description = "약속 시간", type = "string", pattern = "HH:mm", example = "13:30")
        String time,
```
문서만 문제라면 이렇게하면 해결될 거 같아요

그런데 DB에 저장할 때 형식을 지정하지 않고 저장하고 있어 초 까지 저장되서 실제 응답이 문제라면
지금 제리가 작성한대로 적용하거나 `@JsonFormat` 으로 시간 형식을 정해줄 수 도 있겠네용

Reply:
[의견 - 선택]

dummy data 상에서 `HH:mm:ss` 가 문제라면 LocalTime 내에서도 지저분하지만 해결이 가능하긴 한 것 같아요

```java
LocalTime.now().withSecond(0).withNano(0).toString()
```

이렇게 설정하면 HH:mm으로 나오긴 합니다. 제리가 더 낫다고 생각되는 방향으로 수정해도 좋을 것 같아요